11-reading-in-data.Rmd

```diff
@@ -207,4 +207,12 @@ print(names(table(read_dta("data/ven_2007_cy_spa_p.dta")$year)))
 ```
 These are okay so far--let's stick with whatever happens for year/wave until we are at a point where we can actually see data in Tableau.
 
+# convert to long data
```

Comment:
Can we take this section out of this commit and put in a separate branch for #116 ?

Reply:
I created a new branch for #116 and removed this section